gui.py

```diff
@@ -256,23 +230,22 @@ def export_settings(self):
         directory.
         """
         args = {self.user_names[x]:self.vars[x].get() for x in self.user_names}
-        factor_files = {"RTF File": self.rtf_file.get(),
-                        "PTF File": self.ptf_file.get(),
-                        "Airport Growth File": self.airport_growth.get()}
-        args = {**args, **factor_files}
         output_dir = os.path.join(self.vars["tmfs_root"].get(), "Runs", 
-                                  self.vars["tel_year"].get(), "Demand", 
-                                  self.vars["tel_id"].get())
+                                  self.vars["forecast_year"].get(), "Demand", 
+                                  self.vars["forecast_id"].get())
         if not os.path.exists(output_dir):
-            os.makedirs(output_dir)
-            
-            self.log.add_message("Created New Directory - {}".format(
-                    output_dir))
-            
-        file_path = os.path.join(output_dir, "settings.txt")
+            output_dir = ""
+        
+        file_path = filedialog.asksaveasfilename(parent=self.main_frame, 
+                                                 title="Save Settings File",
+                                                 defaultextension=".json",
+                                                 filetypes=[("json files", "*.json")],
+                                                 initialdir=output_dir)

```

Comment:
```suggestion
        file_path = filedialog.asksaveasfilename(
            parent=self.main_frame, title="Save Settings File",
            defaultextension=".json", initialdir=output_dir,
            filetypes=[("JSON files (*.json)", "*.json"), ("All files", "*.*")]
        )     
```

Make file search options more consistent with Windows standards - having an "all files" option is always useful, just in case.